datalake/src/main/java/com/sequenceiq/datalake/entity/DatalakeInstanceGroupName.java

```diff
@@ -0,0 +1,41 @@
+package com.sequenceiq.datalake.entity;
+
+public enum DatalakeInstanceGroupName {
```

Comment:
I've stated this in a previous comment, if you add additional details to this enum, 
it should be renamed to reflect that it's not simply a name, but a collection of properties.
If I'm writing code in a different context, I would never guess that `InstanceGroupName` enum contains details about Scaling rules.
Refactor this enum to have a better naming. DatalakeInstanceGroupDetails or something similar.
Also we have similar logic implemented in https://github.com/hortonworks/cloudbreak/blob/main/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/BlackListedUpScaleRole.java#L9-L9
Might worth if we can make this part of that logic, rather than simply adding yet another control boolean here.